.vscode-test.mjs

```diff
@@ -7,7 +7,7 @@ export default testCli.defineConfig([
 		version: 'insiders',
 		mocha: {
 			ui: 'tdd',
-			timeout: 6000
+			timeout: 60_000
```

Comment:
Did you mean 60000 here? Is it some special syntax? 
Didn't see anything related to `_` here
https://mochajs.org/api/mocha#timeout